app/uk/gov/hmrc/tai/connectors/FileUploadConnector.scala

```diff
@@ -126,7 +126,10 @@ class FileUploadConnector @Inject()(
 
     ahcWSClient
       .url(url)
-      .withHeaders(hc.copy(otherHeaders = Seq("CSRF-token" -> "nocheck")).headers: _*)
+      .addHttpHeaders(
```

Comment:
We are adding to the existing header here, do we still need to add in the Session and Request ID in again?

Reply:
They're missing when verifying the headers in the tests if I delete them from line 131/132